glossary.md

```diff
@@ -19,4 +19,13 @@
 **Service provider**: The entity using Lock Keeper to build a full-featured digital asset manager.
 
 
+## Technical Definitions
+
+**Workflow/Flow**: A set of potentially non-linear steps that accomplish an action on behalf of a user. Completing a workflow requires the execution of one or more protocols. *Example: Generate and store a secret*
+
+**Protocol**: A multi-message cryptographic process that accomplishes a particular task within a workflow. *Example: Generate a secret*
+
+**Session**: An open channel of communication between a client and a server. Used to facilitate the execution of protocols or workflows.
```

Comment:
I think for our purposes they are. Maybe we don't need this definition at all anymore since it seems to only come up in the context of session types.

Reply:
I don't think this will stay true. I would define a session as a logical interaction between (possibly multiple) parties at the application layer, which may involve opening and/or resuming one or more channels (again, possibly with multiple different parties) on the underlying transport layer. To clarify this point further, it seems like we might want separate definitions, one for session and one for channel.

Reply:
Pushed an update with your session definition, a new definition for channels that is basically the old session definition, and I also added a definition for "Message" since it shows up in a lot of definitions and has a particular meaning in code.

"Message" might be a bad term to use for this. The alternate word I've used in code is "Step" but that's a little vague. Do you have any other ideas or is "Message" ok?